src/web/app/pages-admin/admin-accounts-page/admin-accounts-page.component.ts

```diff
@@ -1,29 +1,11 @@
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute, Router } from '@angular/router';
+import { AccountAttributes, AccountInfo, AccountService, CourseAttributes } from '../../../services/account.service';
 import { HttpRequestService } from '../../../services/http-request.service';
 import { NavigationService } from '../../../services/navigation.service';
 import { StatusMessageService } from '../../../services/status-message.service';
 import { ErrorMessageOutput } from '../../error-message-output';
 
-interface CourseAttributes {
```

Comment:
Why this interface get shifted to `account.service`?

Reply:
Because `account.service` utilizes it. I think it is better to put it here since the service part has been abstracted out?

Reply:
Nvm I have moved it back together with AccountInfo